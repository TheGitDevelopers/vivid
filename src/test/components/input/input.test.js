import React from 'react';
import renderer from 'react-test-renderer';
import Input from '../../../components/common/input/input';

describe('Input tests:', () => {
  /* Test mockups */
  const tree = renderer
    .create(<Input />);

  /* All component tests should be listed here */
  it('renders correctly', () => {
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
