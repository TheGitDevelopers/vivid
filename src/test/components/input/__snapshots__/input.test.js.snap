// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input tests: renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#F0F4F8",
        "borderRadius": 6,
        "flexDirection": "row",
        "justifyContent": "center",
        "paddingHorizontal": 20,
        "width": "100%",
      },
      null,
    ]
  }
>
  <TextInput
    allowFontScaling={true}
    autoCompleteType="off"
    editable={true}
    maxLength={300}
    onBlur={[Function]}
    onChangeText={[Function]}
    onFocus={[Function]}
    placeholder=""
    rejectResponderTermination={true}
    secureTextEntry={false}
    style={
      Object {
        "color": "#A6BCD0",
        "fontFamily": "lato-regular",
        "fontSize": 16,
        "padding": 14,
        "width": "90%",
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
</View>
`;
